SH4RE/SH4RE/Messages/Components/MessageView.swift

```diff
@@ -9,77 +9,87 @@ import SwiftUI
 
 struct MessageView: View {
     let message: ChatMessage
-    @State var requestStatus: Int = 0
+    @State var requestStatus: Int = -1
+    
     @State private var requestResponed = false
+    @Binding var showPopUp: Bool;
     
     var body: some View {
-        VStack {
-            if message.fromId == getCurrentUserUid() {
-                HStack {
-                    Spacer()
-                    VStack(alignment: .leading) {
-                        // TODO: add functionality for accepting/denying requests
-                        if (message.isRequest) {
-                            Text("Request sent!").italic()
-                                .foregroundColor(.white)
-                            HStack {
-                                Text("Item: ")
-                                    .foregroundColor(.white)
-                                Text(message.listingTitle).bold()
-                                    .foregroundColor(.white)
-                            }
-                            HStack {
-                                Text("Dates: ")
-                                    .foregroundColor(.white)
-                                Text(message.datesRequested).bold()
+            VStack {
+                if message.fromId == getCurrentUserUid() {
+                    HStack {
+                        Spacer()
+                        VStack(alignment: .leading) {
+                            if (message.isRequest) {
+                                Text("Request sent!").italic()
                                     .foregroundColor(.white)
+                                HStack {
+                                    Text("Item: ")
+                                        .foregroundColor(.white)
+                                    Text(message.listingTitle).bold()
+                                        .foregroundColor(.white)
+                                }
+                                HStack {
+                                    Text("Dates: ")
+                                        .foregroundColor(.white)
+                                    Text(message.datesRequested).bold()
+                                        .foregroundColor(.white)
+                                }
+                                if(requestStatus == 0){
```

Comment:
yeah reading this, I am not realyl sure what the different status' mean, enum would be rlly helpful

Reply:
same as above